### ISSUE TITLE
Shuffle around some elements for better UX

### ISSUE THREAD

**Issue by emmanuelle** [covid19-dash/covid-dashboard#105]
How about we switch the positions of the map and the table so that the table come first and the map comes last? On a big screen it would not change much but on a mobile phone one would have the table first. Happy to make a PR but wanted to discuss this first here.

**Comment by GaelVaroquaux** [comment]
> How about we switch the positions of the map and the table so that the table come first and the map comes last?

Sounds good to me.

**Comment by emdupre** [comment]
Agreed that having the table first would help mobile UX ! @emmanuelle what do you think of moving filtering from the table to a navbar (#103) ? Does this sound do-able with dash ? it has a similar goal in mind, so curious on your thoughts !
